datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -263,7 +263,9 @@ CREATE TABLE dcsa_im_v3_0.booking (
     contract_quotation_reference varchar(35) NULL,
     incoterms varchar(3) NULL REFERENCES dcsa_im_v3_0.incoterms(incoterms_code),
     invoice_payable_at varchar(100) NULL REFERENCES dcsa_im_v3_0.location(id),
-    expected_departure_date timestamp NULL,
+    expected_departure_date date NULL,
+    expected_arrival_date_start date NULL CHECK ((expected_arrival_date_start IS NULL AND expected_arrival_date_end IS NULL) OR expected_arrival_date_start <= expected_arrival_date_end),
+    expected_arrival_date_end date NULL CHECK ((expected_arrival_date_start IS NULL AND expected_arrival_date_end IS NULL) OR expected_arrival_date_start <= expected_arrival_date_end),
```

Comment:
The CHECK is only necessary on one of them.  But I noticed I might have had a bug in my example.  This one says you need BOTH dates or NONE of them.  You cannot provide just the start or the end date.

If you are supposed to provide just one of them, then we need to make the CHECK a bit less strict.

Reply:
I am unsure about this requirement - will have to ask P7 (if endDate can be open or startDate be open...)

Reply:
I removed one of the checks

Reply:
@nt-as reply from P7:

> In my view it should be allowed to provide one date…. i.e. the range should be optional. However at booking request stage the expected arrival date is conditional on expected departure date not being provided.

I have modified the check - please confirm it is correct ;-)